src/generators/ast.ts

```diff
@@ -245,28 +338,37 @@ function parseTypeSpecificFields(raw: JSONSchema7, type: Type): TypeSpecificFiel
   } else if (type === Type.ARRAY) {
     const fields: ArrayTypeFields = { type, items: { type: Type.ANY } };
     if (typeof raw.items !== 'boolean' && raw.items !== undefined) {
```

Comment:
Where do we have `raw.items === "boolean"`. Can you give an example of schema which has this ?